index.html

```
@@ -314,6 +314,11 @@ <h3 class="text-xl font-semibold text-gray-800 dark:text-gray-200">All Contribut
         <div id="contributors-list" class="divide-y divide-gray-200 dark:divide-gray-700">
           <!-- Contributors will be loaded here -->
         </div>
+        <div id="load-more-container" class="p-6 border-t border-gray-200 dark:border-gray-700 text-center hidden">
+          <button id="load-more-btn" class="px-6 py-2 bg-teal-600 text-white font-semibold rounded-lg hover:bg-teal-700 transition-colors">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)